container_pipeline/workers/base.py

```diff
@@ -0,0 +1,63 @@
+import json
+import logging
+import os
+
+from container_pipeline.lib import settings
+from container_pipeline.lib.queue import JobQueue
+
+
+class BaseWorker(object):
+    """Base class for pipeline workers"""
+
+    def __init__(self, logger=None, sub=None, pub=None):
+        self.logger = logger or logging.getLogger('console')
+        self.queue = JobQueue(host=settings.BEANSTALKD_HOST,
+                              port=settings.BEANSTALKD_PORT,
+                              sub=sub, pub=pub, logger=self.logger)
+
+    def handle_job(self, job):
+        """Handle job"""
+        raise NotImplementedError
+
+    def notify_build_failure(self, namespace, notify_email, build_logs_file,
+                             project, jobid, test_tag):
+        """Notify build failure"""
+        data = {}
+        data['action'] = 'notify_user'
+        data['namespace'] = namespace
+        data['build_status'] = False
+        data['notify_email'] = notify_email
+        data['build_logs_file'] = build_logs_file
+        data['project_name'] = project
+        data['job_name'] = jobid
+        data['TEST_TAG'] = test_tag
+        self.queue.put(json.dumps(data), 'master_tube')
```

Comment:
If we're putting failed build data to `master_tube`, why are we creating the tube `failed_build`?